lightswind.css

```
@@ -401,8 +399,8 @@ body {
   border-radius: var(--radius);
   /* Using CSS variable for radius */
   box-shadow:
-    0 10px 15px -3px rgba(0, 0, 0, 0.1),
-    0 4px 6px -2px rgba(0, 0, 0, 0.05);
+    0 10px 15px -3px rgb(0 0 0 / 0.1),
```

Stylelint (17.2.0)

[error] Expected "0.1" to be "10%" (alpha-value-notation)

(alpha-value-notation)